demo/index.html

```diff
@@ -158,6 +158,16 @@
   </article>
 
   <article>
+    <h2>Slider with angular directive inside custom template</h2>
+    <rzslider
+            rz-slider-model="slider_custom_directive_inside_template.minValue"
+            rz-slider-high="slider_custom_directive_inside_template.maxValue"
+            rz-slider-options="slider_custom_directive_inside_template.options"
+            rz-slider-tpl-url="directiveInCustomTemplate.html"
```

Comment:
I don't see the directiveInCustomTemplate.html template, did you forget to include it?